mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/MessageWithoutAttachment.java

```diff
@@ -59,6 +59,10 @@ public MessageWithoutAttachment(MessageId messageId, Date internalDate, Long siz
         this.modSeq = modSeq;
     }
 
+    public RawMessageWithoutAttachment toRawMessageWithoutAttachment() {
```

Comment:
Maybe it would be better to make a interface for message without attachment as we did for Message and SimpleMessage that have attachment.
This will allow to use all method that could take a RawMessageWithoutAttachment (IE a mail with it's mailbox context) take care of message with the context for free.